api/src/main/java/io/opentelemetry/trace/attributes/BooleanAttributeSetter.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.trace.attributes;
+
+import io.opentelemetry.trace.Span;
+import javax.annotation.concurrent.Immutable;
+
+/** Defines the behavior for a span attribute with boolean values. */
+@Immutable
+public class BooleanAttributeSetter {
+
+  private final String attributeKey;
+
+  /**
+   * Constructs an attribute object.
+   *
+   * @param attributeKey the attribute name/key
+   */
+  public BooleanAttributeSetter(String attributeKey) {
+    super();
```

Comment:
why calling this?

Reply:
Ping on this one.

Reply:
@kbrockhoff would be nice to fix this (though personally I wouldn't hold up the PR further just because of this superfluous line).

Reply:
This is fixed. I am not sure why it is showing as outdated.

Reply:
Am I looking to a wrong class https://github.com/open-telemetry/opentelemetry-java/pull/758/files#diff-55ba0be6445e26747fa46c33ffaffdaeR39?